platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/OrganizationMembership.java

```diff
@@ -9,7 +10,7 @@
         @AssociationOverride(name = "key.user", joinColumns = {@JoinColumn(name = "user_id")}),
         @AssociationOverride(name = "key.organization", joinColumns = {@JoinColumn(name = "organization_id")})
 })
-public class OrganizationMembership {
+public class OrganizationMembership implements Serializable {
```

Comment:
Class now implements `Serializable` but is missing field `serialVersionUID` which is strongly recommended.